### DISCUSSION TITLE
Gang playback for multiple windows

### DISCUSSION THREAD

**Discussion by samplue** [iina/iina#4112] [Ideas]
It would be nice to see an option to gang the playhead/playback of two or more windows together. Especially useful if you are comparing two videos and want to play them side-by-side for comparison. CinePlay by Digital Rebellion offers this feature (https://www.digitalrebellion.com/cineplay/) and all major NLEs.

**Comment by low-batt** [comment]
IINA does not provide a nice user interface for playing more than one video side by side, but it is possible to get IINA to do this. See [this comment](https://github.com/iina/iina/issues/3964#issuecomment-1267503453) on issue #3964 for a working example.
